waku/v2/utils/peers.nim

```diff
@@ -69,13 +69,13 @@ proc parseRemotePeerInfo*(address: string): RemotePeerInfo {.raises: [Defect, Va
   let multiAddr = MultiAddress.init(address).tryGet()
 
   var
-
-    ipPart, tcpPart, p2pPart, wsPart, wssPart: MultiAddress
+    nwPart, tcpPart, p2pPart, wsPart, wssPart: MultiAddress
 
   for addrPart in multiAddr.items():
     case addrPart[].protoName()[]
-    of "ip4", "ip6":
-      ipPart = addrPart.tryGet()
+    # All protocols listed here: https://github.com/multiformats/multiaddr/blob/b746a7d014e825221cc3aea6e57a92d78419990f/protocols.csv
+    of "ip4", "ip6", "dns", "dnsaddr", "dns4", "dns6":
+      nwPart = addrPart.tryGet()
```

Comment:
What does `nw` stand for?

Reply:
Network! :) With reference the network OSI layer. I think this part of the code probably needs a rethink, but I had to find some name to replace `ipPart`.

Reply:
You can use `host`, that's the general name used in uri RFC etc